### DISCUSSION TITLE
SMB3 Collectable Cards Suggestion

### DISCUSSION THREAD

**Discussion by vitzie629** [Wohlstand/TheXTech#528] [Ideas]
While SMBX is mostly based on SMAS SMB3 & features the SMB3 Card Roulette exit, there are not any collectible cards on the screen after the player touches the exit resulting either the mushroom, fire flower, or the star. There should be 3 collectible cards for that SMB3 exit just like there was in SMB3 & while 3 cards appear but don’t match exactly resulting a 1-up, 3 mushrooms result a 2-up, 3 fire flowers result a 3-up, & 3 stars result a 5-up! If it was added, the three collectible cards would appear on the bottom right of the screen.

**Comment by Wohlstand** [comment]
Good thing, and this thing is better to be a discussion.

**Reply by vitzie629** [reply]
Thank you!

**Comment by Wohlstand** [comment]
I guess, this concent will be possible to implement via future scripts system.

**Reply by vitzie629** [reply]
Whenever the future scripts system comes out, you should make an example episode for TheXTech featuring the collectible cards & content made with Lua!

**Comment by Shadowblitz16** [comment]
I agree with this goals need to be more useful.

I think the smb3 card system for smb3 roulette would be good.
SMW goal should reward bonus points and allow the developer to set a bonus level.

Maybe goals could be modifyable in the npc options
Maybe smw goals could reward smb3 cards and smb3 roulette could reward bonus points.
Maybe there could even be a option to let users set what it rewards and how much...

The following types could be pickable in the npc options...
- type
- - roulette card
- - bonus points
- - score
- int:amount
- int:max
- level:scene on max
- int:additional point per amount
- bool:reward only at max

Hud might need to be updated to fit the bonus points and roulette cards.
Maybe they would only show when in a certain radius of the goal
